bots/naughty/rhel-8/12539-libvirtd-crash-domain-destroy

```diff
@@ -0,0 +1,3 @@
+Traceback (most recent call last):
+  *
+  File *machineslib.py*, line *, in deleteVm
```

Comment:
This is too generic, it could easily catch pretty much *any* other error. Could you match on this:
```
  File "test/verify/machineslib.py", line *, in createTest
    runner.deleteVm(dialog) \
  File "test/verify/machineslib.py", line *, in deleteVm
    b.wait_not_present("#vm-{0}-delete-modal-dialog".format(dialog.name))
```

and if possible, append this:
```
*
wait_js_cond(!ph_is_present("#vm-subVmTestCreate21-delete-modal-dialog"))
```

So that you only catch this particular test?